libs/utils/compare-versions.js

```diff
@@ -0,0 +1,39 @@
+const semver = require('semver')
+
+module.exports = (v1, v2) => {
+  const validV1 = semver.valid(v1)
+  const validV2 = semver.valid(v2)
+
+  const isV1Valid = !!validV1
+  const isV2Valid = !!validV2
+
+  if (!isV1Valid && !isV2Valid) {
+    const numberVersionRegex = /^\d/
```

Comment:
Seems this can be omitted, since `semver.coerce()` could turn `numberVersion` into valid semver.